checks.d/mesos_master.py

```diff
@@ -190,7 +194,13 @@ def check(self, instance):
         default_timeout = self.init_config.get('default_timeout', 5)
         timeout = float(instance.get('timeout', default_timeout))
 
-        state_metrics = self._check_leadership(url, timeout)
+        no_ssl = _is_affirmative(instance.get('disable_ssl_validation', False))
+        ssl_verify = False if no_ssl else True
+        parsed_url = urlparse(url)
+        if no_ssl and parsed_url.scheme == 'https':
+            self.log.warning('Skipping SSL cert validation for %s based on configuration.' % url)
```

Comment:
I could add a `ignore_ssl_warning` setting like in the http_check (which has the exact same behavior)? It's something important to remember so we should probably log something.